### PR TITLE
Improve(code): Improved the code

### PR DESCRIPTION
**Pull request by RajputUsman**
Fixes: #19
If this got merged kindly label it as `hacktoberfest-accepted`.

### CODE REVIEW

File: load_stations.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
     station_data = {x["name"]: x["value"] for x in sd.data}
 
     station_key = f"station:{str(station.name).lower()}"
-    pipeline.hmset(station_key, station_data)
+    pipeline.hset(station_key, station_data)
```
Comment (simonprickett) [inline load_stations.py:39]: Did you test run this?  It errors:

```
$ python load_stations.py
Traceback (most recent call last):
  File "/Users/simonprickett/source/github/introducing-the-geosearch-command/load_stations.py", line 44, in <module>
    responses = pipeline.execute()
  File "/Users/simonprickett/source/github/introducing-the-geosearch-command/venv/lib/python3.10/site-packages/redis/client.py", line 4019, in execute
    return execute(conn, stack, raise_on_error)
  File "/Users/simonprickett/source/github/introducing-the-geosearch-command/venv/lib/python3.10/site-packages/redis/client.py", line 3949, in _execute_pipeline
    all_cmds = connection.pack_commands([args for args, _ in commands])
  File "/Users/simonprickett/source/github/introducing-the-geosearch-command/venv/lib/python3.10/site-packages/redis/connection.py", line 801, in pack_commands
    for chunk in self.pack_command(*cmd):
  File "/Users/simonprickett/source/github/introducing-the-geosearch-command/venv/lib/python3.10/site-packages/redis/connection.py", line 775, in pack_command
    for arg in imap(self.encoder.encode, args):
  File "/Users/simonprickett/source/github/introducing-the-geosearch-command/venv/lib/python3.10/site-packages/redis/connection.py", line 119, in encode
    raise DataError("Invalid input of type: '%s'. Convert to a "
redis.exceptions.DataError: Invalid input of type: 'dict'. Convert to a bytes, string, int or float first.
```

**Review by simonprickett — CHANGES_REQUESTED** [review]

**Comment by SuzeShardlow** [comment]
@RajputUsman please see @simonprickett's comments in his review above ^^ :)

**Comment by SuzeShardlow** [comment]
@RajputUsman please let me know if you are still working on this.  If not, I will close the PR.  Thanks!

**Comment by RajputUsman** [comment]
Thanks, I tried to resolve with docker, but it didn't work out.

**Pull request closed without merging**